### DISCUSSION TITLE
有计划支持React吗？

### DISCUSSION THREAD

**Discussion by pincman** [tusen-ai/naive-ui#1817] [General]
请问这个ui库有没有计划支持react呢？

**Comment by 07akioni** [comment]
没有，工程量太大了，做不到

**Reply by 07akioni** [reply]
其实如果我们真的有这个资源的话，相比于复刻一遍这个库，不如去探索一下 ui 库的新边界

**Comment by pincman** [comment]
@07akioni 写react目前非常尴尬,，就一个antd可以用,搞个暗黑还要hack，vue的vant,nutui,vuetify和你这个都很不错，对设计不行的人很是友好

**Reply by 07akioni** [reply]
我估计 antd 5.0 版本之后会好一些，他们历史包袱比较重，功能不好加

**Reply by 07akioni** [reply]
react 如果不要太复杂的 case 的话其实也有一些其他的选择，material ui、chakara

**Reply by pincman** [reply]
是的，但是都没你这个好用，哈哈，material ui写css是爽，但是做好看需要很好的设计水平，本人这种主后端狗搞不起来😄
